src/dune_rules/install_rules.ml

```diff
@@ -268,8 +268,9 @@ end = struct
     in
     let+ execs = lib_ppxs ctx ~scope ~lib in
     let install_c_headers =
-      List.map lib.install_c_headers ~f:(fun base ->
-          Path.Build.relative dir (base ^ Foreign_language.header_extension))
+      List.rev_map lib.install_c_headers ~f:(fun (loc, base) ->
```

Comment:
Since you've changed this from a map to a rev_map this won't affect anything right?

Reply:
makes no difference here